src/ophyd_async/epics/_backend/_p4p.py

```diff
@@ -141,6 +140,20 @@ def descriptor(self, source: str, value) -> Descriptor:
         return dict(source=source, dtype="object", shape=[])  # type: ignore
 
 
+class PvaDictConverter(PvaConverter):
+    def reading(self, value):
+        value = value.todict()
+        ts = time.time()
```

Comment:
So this isn't coming from the signal?

Reply:
Nah, the pva table doesn't have a timestamp (or an error state). We just add one ourselves from when we read it.